src/NzbDrone.Common/Http/Dispatchers/ManagedHttpDispatcher.cs

```diff
@@ -252,13 +258,22 @@ private CredentialCache GetCredentialCache()
         private static bool HasRoutableIPv4Address()
         {
             // Get all IPv4 addresses from all interfaces and return true if there are any with non-loopback addresses
-            var networkInterfaces = NetworkInterface.GetAllNetworkInterfaces();
+            try
+            {
+                var networkInterfaces = NetworkInterface.GetAllNetworkInterfaces();
 
-            return networkInterfaces.Any(ni =>
-                ni.OperationalStatus == OperationalStatus.Up &&
-                ni.GetIPProperties().UnicastAddresses.Any(ip =>
-                    ip.Address.AddressFamily == AddressFamily.InterNetwork &&
-                    !IPAddress.IsLoopback(ip.Address)));
+                return networkInterfaces.Any(ni =>
+                    ni.OperationalStatus == OperationalStatus.Up &&
+                    ni.GetIPProperties().UnicastAddresses.Any(ip =>
+                        ip.Address.AddressFamily == AddressFamily.InterNetwork &&
+                        !IPAddress.IsLoopback(ip.Address)));
+            }
+            catch (Exception e)
+            {
+                _logger.Debug("Caught exception while GetAllNetworkInterfaces assuming IPv4 connectivity");
+                _logger.Trace("HasRoutableIPv4Address: " + e.Message);
```

Comment:
I don't think we need to log this separately, could append to the debug message.